### PR TITLE
Chirp Lab completed 

### PR DESCRIPTION
**Pull request by cesrebol**

### CODE REVIEW

**Review by iammattwalsh — CHANGES_REQUESTED** [review]
This is great so far, but it's missing the "profile view" (a list of all chirps by a user). Add that and you'll be set!

**Comment by cesrebol** [comment]
@MattDWalsh Hey Matt! I made the changes you have requested. Thanks again!

File: code/cesar/django/lab_03/chirp/chirp/urls.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('accounts/', include('users.urls')),
+    path('users/', include('users.url')),
```
Comment (iammattwalsh) [inline code/cesar/django/lab_03/chirp/chirp/urls.py]: Check this line!

**Review by iammattwalsh — CHANGES_REQUESTED** [review]
You are literally ONE character away from this being done! Check line 24 in your _project_ urls.py.

**Comment by cesrebol** [comment]
@Gage-Lieble Hi Gage, I have fixed the type Matt found. Could you check this out when you get the chance? Thanks!

**Comment by Gage-Lieble** [comment]
Looks good!

**Pull request merged**